kakaemma/My-Diary-React
# #162339702 add loader to the application

#### What does this PR do?
This PR adds add a loader to the application.

### How should this be manually tested?
- clone the repository with `git clone`
- create another branch off the current one with `git checkout -b <your_branch_name>` 
- run `git pull origin ch-add-loader-to-the-application `
- Run `npm run mydiary` to install all dependencies and start the application
- Enter the URL `http://localhost:3000/` in your browser.
- click login after entering your user details.
- A loader should be displayed when the button is clicked

#### What are the relevant pivotal tracker stories?
[#162339702](https://www.pivotaltracker.com/story/show/162339702)

#### Any background context you want to add?
N/A

## Checklist: I completed these to help reviewers
